spring-cloud-data-core/src/main/java/org/springframework/cloud/data/core/dsl/DSLMessage.java

```diff
@@ -59,6 +59,7 @@
 	EXPECTED_CHANNEL_QUALIFIER(ERROR, 120, "expected channel reference '':<channel>'' but found ''{0}''"), //
 	EXPECTED_CHANNEL_NAME(ERROR, 121, "expected channel name but found ''{0}''"), //
 	ILLEGAL_STREAM_NAME(ERROR, 122, "illegal name for a stream ''{0}''"), //
+	ILLEGAL_TASK_NAME(ERROR, 122, "illegal name for a task ''{0}''"), //
```

Comment:
I think the code `122` needs to change as it is already used by `ILLEGAL_STREAM_NAME`